javascript/javascript2/week1/main/index.js

```diff
@@ -0,0 +1,20 @@
+//Netlify deployment: https://vocal-cannoli-85e9ae.netlify.app
+
+const div = document.createElement('div');
+div.className = 'card';
+document.body.appendChild(div);
+
+div.addEventListener("click", function(){
+    if(div.className === 'card'){
```

Comment:
Using [classList](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList) instead of className would make your code more manageable here.